src/pages/host/hostDetail/detail.jsx

```diff
@@ -0,0 +1,64 @@
+import { Link, useParams, Outlet, useLoaderData } from "react-router-dom";
+import { HiArrowLeft } from "react-icons/hi";
+import HostDetailNav from "../../../components/host/hostDetailNav";
+import { getVans } from "../../../api";
+import { loader as loadLogOut }  from "../../authentication";
+
+export async function loader({ request }) {
+  return { vans: await getVans(), prevPath: loadLogOut({ request }) };
+}
+
+export default function HostVanDetail() {
+  const hostList=useLoaderData();
+  const listId = useParams();
+  const item = hostList.vans[listId.id];
+  console.log(hostList)
+  const loading = false;
+
+  return loading ? (
+    <div className="text-center py-32">loading...</div>
+  ) : (
```

Comment:
There is not reason to use the ternary operator as loading will always be false, so you can just return.

Reply:
DONE :heavy_check_mark: